sdk/servicebus/Microsoft.Azure.ServiceBus/tests/QueueSessionTests.cs

```diff
@@ -16,14 +16,14 @@ public sealed class QueueSessionTests
         {
             // Expected structure: { usePartitionedQueue, useSessionQueue }
             new object[] { false, true },
-            new object[] { true, true }
+            //new object[] { true, true }
         };
 
         [Theory]
         [MemberData(nameof(TestPermutations))]
         [LiveTest]
         [DisplayTestMethodName]
-        public async Task SessionTest(bool partitioned, bool sessionEnabled)
+        public async Task JESSE_SessionTest(bool partitioned, bool sessionEnabled)
```

Comment:
You might want to remove the JESSE prefix :).

Reply:
I'm of the opinion that everything should start with JESSE.    Um...  whoops.  

![](https://media2.giphy.com/media/3ohzdYJK1wAdPWVk88/giphy.gif)